examples/tv-casting-app/linux/main.cpp

```diff
@@ -0,0 +1,141 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/server/Mdns.h>
+#include <app/server/Server.h>
+#include <controller/CHIPCommissionableNodeController.h>
+#include <platform/CHIPDeviceLayer.h>
+#include <platform/ConfigurationManager.h>
+#include <system/SystemLayer.h>
+#include <transport/raw/PeerAddress.h>
+
+using namespace chip;
+using namespace chip::Controller;
+
+#define TV_DEVICE_TYPE 35
```

Comment:
please use C++ constants: `constexpr int kTvDeviceType = 35`. This does type checks and compiler will also warn of unused constants if this becomes obsolete.

Should this constant not come from some other include?